tox.ini

```diff
@@ -11,40 +11,48 @@ setenv =
 whitelist_externals =
     make
     npm
+    bash
 
 [testenv:es]
 commands =
     make -f Makefile-docker update_deps
+    bash {toxinidir}/locale/compile-mo.sh {toxinidir}/locale/
```

Comment:
unfortunately yes, since we create a new travis job for every test-environment.

I am already working on a follow-up that marks locale-relevant tests so that we can create a new environment, similar to our `es_test` marker and run them independently.